src/category_theory/opposites.lean

```diff
@@ -98,6 +98,8 @@ instance category.opposite : category.{v₁} Cᵒᵖ :=
 @[simp] lemma unop_comp {X Y Z : Cᵒᵖ} {f : X ⟶ Y} {g : Y ⟶ Z} :
   (f ≫ g).unop = g.unop ≫ f.unop := rfl
 @[simp] lemma unop_id {X : Cᵒᵖ} : (𝟙 X).unop = 𝟙 (unop X) := rfl
+@[simp] lemma id_op_unop {X : C} : (𝟙 (op X)).unop = 𝟙 X := rfl
```

Comment:
Shouldn't this be called `unop_id_op`?

Reply:
Sure. I'm not using them anymore, but I'll leave them in, I think.